### PR TITLE
fix reference frame dof cap out of bounds

### PR DESCRIPTION
**Pull request by edaniels**

### CODE REVIEW

File: robot/web/web.go [hunk 1]
```diff
@@ -395,7 +395,6 @@ func (svc *webService) StartModule(ctx context.Context) error {
 	svc.activeBackgroundWorkers.Add(1)
 	utils.PanicCapturingGo(func() {
 		defer svc.activeBackgroundWorkers.Done()
-		defer utils.UncheckedErrorFunc(func() error { return os.Remove(addr) })
```
Comment (edaniels) [inline robot/web/web.go:398]: plus a driveby

File: referenceframe/model.go [hunk 2]
```diff
@@ -160,7 +160,7 @@ func (m *SimpleModel) DoF() []Limit {
 	}
 	m.lock.RUnlock()
 
-	limits := make([]Limit, 0, len(m.OrdTransforms)-1)
```
Comment (edaniels) [inline referenceframe/model.go:163]: this is a golang no no

**Review by Otterverse — APPROVED** [review]
LGTM

**Review by maximpertsov — APPROVED** [review]
